tests/product_review.test.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const db = require('APP/db');
+const Product_Review = require('APP/db/models/product_review');
+const expect = require('chai').expect;
+
+describe('Product Model', () => {
```

Comment:
Good testing guys. It seems you are only testing for validation errors but not that the database actually contains data that you created an instance with. Shouldn't you also test that the build was successful and actually contains the data you fed it?